js/whitebit.js

```diff
@@ -361,103 +401,68 @@ module.exports = class whitebit extends Exchange {
 
     parseTicker (ticker, market = undefined) {
         //
-        // fetchTicker
-        //
-        //     {
-        //         "bid":"0.021979",
-        //         "ask":"0.021996",
-        //         "open":"0.02182",
-        //         "high":"0.022039",
-        //         "low":"0.02161",
-        //         "last":"0.021987",
-        //         "volume":"2810.267",
-        //         "deal":"61.383565474",
-        //         "change":"0.76",
-        //     }
-        //
-        // fetchTickers v1
-        //
-        //     {
-        //         "at":1571022144,
-        //         "ticker": {
-        //             "bid":"0.022024",
-        //             "ask":"0.022042",
-        //             "low":"0.02161",
-        //             "high":"0.022062",
-        //             "last":"0.022036",
-        //             "vol":"2813.503",
-        //             "deal":"61.457279261",
-        //             "change":"0.95"
-        //         }
-        //     }
+        // "BCH_RUB":{
+        //     "base_id":1831,
+        //     "quote_id":0,
+        //     "last_price":"32830.21",
+        //     "quote_volume":"1494659.8024096",
+        //     "base_volume":"46.1083",
+        //     "isFrozen":false,
+        //     "change":"2.12" // in percent
+        //  },
         //
-        const timestamp = this.safeTimestamp (ticker, 'at', this.milliseconds ());
-        ticker = this.safeValue (ticker, 'ticker', ticker);
         let symbol = undefined;
         if (market !== undefined) {
             symbol = market['symbol'];
         }
-        const last = this.safeNumber (ticker, 'last');
-        const percentage = this.safeNumber (ticker, 'change');
-        let change = undefined;
-        if (percentage !== undefined) {
-            change = this.numberToString (percentage * 0.01);
-        }
+        const last = this.safeNumber (ticker, 'last_price');
+        const percentage = this.safeNumber (ticker, 'change') * 0.01;
+        const change = last * percentage;
+        const initial = last - change;
```

Comment:
Let's not calculate lines 418-421, they must be done in `safeTicker`.
We should `return this.safeTicker ({ ... })` – see how fetchTicker / parseTicker is implemented across the other files.

Reply:
@kroitor lines 418 (last) and 419 (percentage), I'm not calculating them only parsing from the raw ticker, those I can keep right?

Reply:
Yep.